code/modules/vehicles/cars/car.dm

```diff
@@ -93,3 +96,116 @@
 		M.visible_message("<span class='warning'>[M] is forced into \the [src]!</span>")
 	M.forceMove(src)
 	add_occupant(M, VEHICLE_CONTROL_KIDNAPPED)
+
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedestrian-focused station, and now all these cars are ruining it!" //ok boomer
+	icon_state = "car"
+	max_integrity = 300
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	enter_delay = 50 //No speed escapes
+	max_occupants = 5
+	movedelay = 1.15
+	material_flags = MATERIAL_AFFECT_STATISTICS | MATERIAL_COLOR | MATERIAL_ADD_PREFIX
+	var/obj/item/card/id/linked_id = null
+	var/can_drive = FALSE
+
+
+/obj/vehicle/sealed/car/civ/Initialize()
+	var/datum/material/M = pick(subtypesof(/datum/material))
+	custom_materials = list()
+	custom_materials[M] = 10000
+	. = ..()
+
+
+/obj/vehicle/sealed/car/civ/generate_actions()
+	. = ..()
+	initialize_controller_action_type(/datum/action/vehicle/sealed/horn, VEHICLE_CONTROL_DRIVE)
+
+/obj/vehicle/sealed/car/civ/Cross(atom/movable/AM)
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE IN FRONT OF MY CAR YA FUCKIN' KNOB!!", "HEY! I'M DRIVIN' HERE!!", "YOU ARE RUINING MY SUSPENSION, CUNT!"), forced="car crash")
```

Comment:
No sexist slurs on our family friendly Github repo please

Reply:
shut the fuck up goof

Reply:
if someone dents my car's paint I will call them whatever word i want

Reply:
it's obviously cunt in the non-american sense

Reply:
doesn't matter, this is the github, you need to abide by Github's ToS @Qustinnus